website/blog/2023-04-19-running-a-local-kubernetes-cluster-with-podman-desktop.md

```diff
@@ -0,0 +1,272 @@
+
+1. Go to **Settings > Resources**
+1. In the **Kind** tile, click on the **Create new ...** button.
+   1. **Name**: type `kind-cluster`.
+   1. **Provider Type**: select `podman`.
+   1. **HTTP Port**: select `9090`.
+   1. **HTTPS Port**: select `9443`.
+   1. **Setup an ingress controller**: `Enabled`
+   1. Click the **Create** button.
+1. After successful creation, click on the **Go back to resources** button
+1. In the **Podman Desktop** tray, open the **Kubernetes** menu: you can set the context to your Kind cluster: `kind-kind-cluster`.
+
+Now you can start working with containers, and your local Kubernetes cluster.
+
+#### Additional resources
+
+- [Creating a local Kind-powered Kubernetes cluster](docs/kubernetes/kind/creating-a-kind-cluster)
+
+## Starting the Redis leader
+
+The guestbook application uses Redis to store its data.
+
+With Podman Desktop, you can prepare the Redis leader image and container on your local container engine, and deploy the results to a Kubernetes pod and service.
+This is functionally equivalent to the `redis-leader` deployment that the Kubernetes example propose.
+
+#### Procedure
+
+1. Open **<icon icon="fa-solid fa-cloud" size="lg" /> Images > <icon icon="fa-solid fa-arrow-circle-down" size="lg" /> Pull an image**.
+   1. **Image to Pull**: type `docker.io/redis:6.0.5`
+   1. Click **Pull image** to pull the image to your container engine local image registry.
+   1. Click **Done** to get back to the images list.
+1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search images...**: type `redis:6.0.5` to find the image.
+1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-ellipsis-v" size="lg" /> Push image to Kind cluster**.
+1. Click **<icon icon="fa-solid fa-play" size="lg" />** to open the **Create a container from image** dialog.
+   1. **Container name**: type `leader`,
+   1. **Local port for 6379/tcp**: `6379`.
+   1. Click **<icon icon="fa-solid fa-play" size="lg" /> Start Container** to start the container in your container engine.
+1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search containers...**: type `leader` to find the running container.
+1. Click **<icon icon="fa-solid fa-stop" size="lg" />** to stop the container, and leave the `6379` port available for the Redis follower container.
+
+1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-rocket" size="lg" /> Deploy to Kubernetes** to open the **Deploy generated pod to Kubernetes** screen.
+   1. **Pod Name**: type `redis-leader`.
+   1. **Use Kubernetes Services**: select **Replace .hostPort exposure on containers by Services. It is the recommended way to expose ports, as a cluster policy might prevent to use hostPort.**
+   1. **Kubernetes namespaces**: select `guestbook`.
+   1. Click **<icon icon="fa-solid fa-rocket" size="lg" /> Deploy**.
+   1. Wait for the pod to reach the state: **Phase: Running**.
```

Comment:
With the podman desktop binary built from main branch (0.15.0-next as of today) I am not able to get any pods. Seems like some regression. Cannot continue.

Reply:
Did you try to refresh the page? (Ctrl + R)